keras_core/layers/preprocessing/discretization.py

```diff
@@ -17,13 +17,9 @@ class Discretization(Layer):
     contiguous ranges and output an integer index indicating which range each
     element was placed in.
 
-    **Note:** This layer uses TensorFlow internally. It cannot
-    be used as part of the compiled computation graph of a model with
-    any backend other than TensorFlow.
-    It can however be used with any backend when running eagerly.
-    It can also always be used as part of an input preprocessing pipeline
-    with any backend (outside the model itself), which is how we recommend
-    to use this layer.
+    This layer can also always be used as part of an input preprocessing
+    pipeline with any backend (outside the model itself), which is how we
+    recommend to use this layer.
 
     **Note:** This layer is safe to use inside a `tf.data` pipeline
```

Comment:
To make tf.data work you need to inherit from `TFDataLayer`

Reply:
done